person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :age, :name
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] Let's add id that should only be able to be retrieved so only the Getter is defined.
`attr_reader :id`